spec/forms/partner_details_form_spec.rb

```diff
@@ -17,7 +16,6 @@
   end
 
   it "has some long-form copy and questions about employment" do
```

Comment:
this test no longer checks long form content so can be renamed

Reply:
Good spot! Fixing now.